### ISSUE TITLE
Normalizer will normalize the test data with mean and std of test unless it is used on train data before

### ISSUE THREAD

**Issue by Wensi-Tang** [gzerveas/mvts_transformer#26]

**Comment by gzerveas** [comment]
Hello, yes, the assumption is that the model will first be trained, or otherwise training set statistics will be provided, before proceeding to evaluation on the test set.

**Comment by Wensi-Tang** [comment]
Thanks so much! You are doing it well, but I just think this might be dangerous for someone to reproduce the result.

**Comment by gzerveas** [comment]
This is a good point. Maybe it would be best to add a warning, in case someone uses `--test_only` option but does not use the `--norm_from` option.
